### DISCUSSION TITLE
How to reset password

### DISCUSSION THREAD

**Discussion by pedraamn** [apache/devlake#6242] [Q&A] [answered]
Just for context I am pulling up the grafana by port-forwarding my pod locally. I have installed Devlake on EKS using helm.

The issue I am running into is when I try the default username: admin and password: admin I get an incorrect credentials response. Has anyone seen this before? Or does someone know how I can reset the password even though I don't official have an account?

**Comment by d4x1** [comment]
@ZhangNing10 Can you help to solve it ?

**Reply by ZhangNing10** [reply]
Hi @pedraamn , you can get grafana initial password via decoding the password from grafana secret

**Reply by pedraamn** [reply]
Hi @ZhangNing10 thankyou, how can I decode it? I am able to see the secret info, but not sure how to decode it. Do you know what encoding method is used?

**Reply by ZhangNing10** [reply, marked as answer]
run `kubectl get secret -n <your namespace> ` to get grafana secret, and then `kubectl get secret -n <your namespace>  <grafana secret name> -oyaml`, and then `echo <password encode string> |base64 --decode`

**Reply by pedraamn** [reply]
thankyou very much this worked @ZhangNing10

**Comment by Rajmohan9524** [comment]
Hi I am user in loacl Devlake 

I am forgot Grafana Password, not able to Reset Could you please help on this

**Reply by klesh** [reply]
@Rajmohan9524 Try google, https://shashanksrivastava.medium.com/how-to-reset-grafana-admin-password-e85d54f7b62a

**Reply by Rajmohan9524** [reply]
@klesh Need coments for Docker
